__tests__/app/message-table.test.tsx

```diff
@@ -1,9 +1,16 @@
 import { fireEvent, render, screen } from "@testing-library/react";
 import { MessageTable } from "@/app/message-table";
-import { experimentColumns, RecipeOrBranchInfo } from "@/app/columns";
+import { experimentColumns, fxmsMessageColumns, FxMSMessageInfo, RecipeInfo, RecipeOrBranchInfo } from "@/app/columns";
 import { ExperimentFakes } from "@/__tests__/ExperimentFakes.mjs";
+import { NimbusRecipeCollection } from "@/lib/nimbusRecipeCollection";
 import { NimbusRecipe } from "@/lib/nimbusRecipe";
 
+jest.mock("../../lib/sdk")
+
+global.structuredClone = jest.fn((val) => {
```

Comment:
This seems to be an issues with jsdom.  To avoid having to define this all over the place, try using something like the solution shown in https://github.com/jsdom/jsdom/issues/3363#issuecomment-1575996129.

Reply:
Thanks